instat/ucrCore.vb

```diff
@@ -280,7 +280,7 @@ Public Class ucrCore
                 If bAdd Then
                     lstAllRCodes(i).AddParameter(lstAllRParameters(i))
                 Else
-                    lstAllRCodes(i).RemoveParameter(lstAllRParameters(i))
+                    lstAllRCodes(i).RemoveParameterByName(lstAllRParameters(i).strArgumentName)

```

Comment:
Great, it removes the parameter correctly now, though on second and subsequent runs it fails to pass the respective values, intrestingly..`last_graph <- sjPlot::plot_grid(x=sjPlot::sjplot(data=InstatDataObject$get_columns_from_data(force_as_data_frame=TRUE, col_names=c("Size","Fert"), data_name="survey"), weight.by=, auto.group=, coord.flip=FALSE))` checking on this as well.